documents/src/navigation.md

```diff
@@ -89,7 +89,9 @@ type: nav
     - [Overview](./accessibility/overview)
   - [Internationalization (i18n)](./intl/internationalization)
   - [Theme Switching](./guides/theme-switching)
-
+- Building Components
+  - Utilities
+  * [Cache](./utils/cache)
```

Comment:
Remove from nav for now

Reply:
We can add it back when we continue with `efx-element` documentation

Reply:
Updated by following your suggest. I remove the hole change in this file